src/ui/map.js

```diff
@@ -434,6 +450,7 @@ class Map extends Camera {
         return this;
 
     }
+
```

Comment:
everywhere else had a whitespace between methods, so I added one here to be consistent.